controllers/apps/transformer_component_account.go

```diff
@@ -122,11 +122,13 @@ func (t *componentAccountTransformer) getPasswordFromSecret(ctx graph.TransformC
 }
 
 func (t *componentAccountTransformer) buildPassword(ctx *componentTransformContext, account appsv1alpha1.SystemAccount) []byte {
-	if !account.InitAccount {
-		return t.generatePassword(account)
+	password := ""
+	if account.InitAccount {
+		password = factory.GetRestorePassword(ctx.SynthesizeComponent)
```

Comment:
initAccount also can restore from` factory.GetRestoreSystemAccountPassword(ctx.SynthesizeComponent, account)`, only need a compatibility processing and add the note